akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -250,7 +250,7 @@ private[stream] object ActorGraphInterpreter {
           rejectAdditionalSubscriber(subscriber, s"${Logging.simpleName(this)}")
       }
 
-    def exposedPublisher(publisher: ActorPublisher[Any]): Unit = {
+    private[akka] def exposedPublisher(publisher: ActorPublisher[Any]): Unit = {
```

Comment:
ah, this was a test balloon, will remove: the reason is https://issues.scala-lang.org/browse/SI-9490